src/wr_control_drive_arm/src/JointStatePublisher.cpp

```diff
@@ -58,8 +58,12 @@ void publishJointStates(const ros::TimerEvent &event) {
 }
 
 auto getEncoderConfigFromParams(const XmlRpcValue &params, const std::string &jointName) -> EncoderConfiguration {
-    return {.countsPerRotation = static_cast<int32_t>(params[jointName]["counts_per_rotation"]),
-            .offset = static_cast<int32_t>(params[jointName]["offset"])};
+    return {.countsPerRotation = static_cast<int32_t>(params[jointName]["encoder_parameters"]["counts_per_rotation"]),
+            .offset = static_cast<int32_t>(params[jointName]["encoder_parameters"]["offset"])};
+}
+
+auto getMotorConfigFromParams(const XmlRpcValue &params, const std::string &jointName) -> MotorConfiguration {
+    return {.gearRatio = static_cast<double>(params[jointName]["motor_configurations"]["gear_ratio"])};
 }
```

Comment:
SUGGESTION: Since this is defined in two places the same way, maybe it makes sense to abstract these functions to their own header files and link the implementation in CMake.

Reply:
Won't do (for now lol)